README.md

```diff
@@ -189,6 +189,12 @@ cfg_condition = "mycond"
     name = "baseline-position"
     version = "3.10"
     ignore = true
+    [[object.property]]
+    name = "events"
+    # generate only `connect_property_notify_event`, without `get_property_events` and `set_property_events`
```

Comment:
Shouldn't it be `connect_property_notify_eventS`?

Reply:
Strange, but is not:
```
    fn connect_property_notify_event<F: Fn(&Self, &gdk::EventProperty) -> Inhibit + 'static>(&self, f: F) -> SignalHandlerId {
        unsafe {
            let f: Box_<Box_<Fn(&Self, &gdk::EventProperty) -> Inhibit + 'static>> = Box_::new(Box_::new(f));
            connect(self.to_glib_none().0, "property-notify-event",
                transmute(property_notify_event_trampoline::<Self> as usize), Box_::into_raw(f) as *mut _)
        }
    }

```

Reply:
Let's call it dark magic haha.

Reply:
You was right, I looked to wrong function: `connect_property_events_notify`

Reply:
Ah now it seems more logical! (And explain why I couldn't find it...)

Reply:
> IMHO needed other name instead "object.property.generate", maybe for `PropertyGenerateFlags` too.

generate seems fine to me

Reply:
Oh sorry, forgot to answer this question: seems fine to me as well.